portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/openidconnect/dependencies/com.liferay.portal.security.sso.openid.connect.internal.configuration.OpenIdConnectProviderConfiguration~PingOne.config

```diff
@@ -4,8 +4,8 @@ discoveryEndPointCacheInMillis=L"360000"
 idTokenSigningAlgValues=["RS256"]
 issuerURL=""
 jwksURI=""
-openIdConnectClientId="dedf1d45-02c3-49d4-bc68-498b608dc7c4"
-openIdConnectClientSecret="qPeVJYSc9xugylKo7FnhOpbF7onTKtEsSDQimFU4U1oN5HQTd7DG1uugT~6gLN6Q"
+openIdConnectClientId="${openid.provider.pingone.client.id}"
+openIdConnectClientSecret="${openid.provider.pingone.client.secret}"
```

Comment:
Should be openid.provider.pingone.client.password: https://github.com/liferay/liferay-jenkins-ee/blob/master/commands/build.properties#L1114